source/develop/release-management/features/network/ovirt-lldp-labeler.html.md

```diff
@@ -0,0 +1,127 @@
+to automate some network related tasks. According to configuration reported 
+via LLDP from peer switch, the labaler is capable of labeling host network 
+interface or bond interfaces together. 
+
+## Prerequisites
+
+The labler has to be capable of reaching desired engine REST API and the 
+`python-ovirt-engine-sdk4` in version greater than `4.2` has to be installed.
+
+
+## Limitations
+
+Currently only the Juniper switches supports reporting of the needed parameters.
+The required parameters are:
+* `Port VLAN`
+* `Link Aggregation`
+
+## Implementation
+
+Overall architecture:
+![](/images/wiki/ovirt-lldp-labeler-architecture.png)
+
+### Service
+
+The labeler service is done through timers availble in systemd. The timer calls
+target which executes the cli tool. The delay is one hour by default and can be 
+changed in the timer specification.
+
+### Labeler
+
+The tool itself is divided into two parts. The labeling part and the bonding 
+part, each can be enabled or disabled in configuration file. 
+
+Each step described below is done for each host, that is part of the cluster 
+specified in the configuration file.
+
+- The bonding part starts by fetching all LLDP availble for interfaces that are 
+not part of any existing bond already. The labeler is not trying to modify 
+existing bonds, just to create new ones. The LLDP information is then searched 
+for `Link Aggregation` capabilites. The interfaces, that are capable of 
+aggregation, are grouped together by `Aggregation Link ID`. Before creating 
+bond, the interface with attached `ovirtmgmt` network is filtered out, as we 
+don't want to risk losing connectivity with the host. Also rules about which 
+networks can be attached to interface are applied. After creating bond 
+candidates the last step is to re-attach all networks from slaves to bond 
+candidate. All network configurations made by the labeler are marked as 
+permanent so host restart won't wipe them.
+
+
+- The labeling part fetches LLDP from all non-vlan interfaces and bond slaves.
+The labeler search every interface LLDP information for `VLAN ID` tlvs. For 
+every VLAN tlv, the labeler creates label candidate, in format of 
+`lldp_vlan_$(VLAN-ID)`, where `$(VLAN-ID)` is substituted with VLAN id from 
+the reported LLDP information. It is possible to have multiple VLAN on 
+the interface, in this case the interface ends up with multiple labels. 
+
+
+## Configuration
+
+The labeler consists of three main configuration files.
+Full configuration documentation is available in the [project readme](https://github.com/oVirt/ovirt-lldp-labeler/blob/master/README.adoc#configuration).
+
+First two files are usually located under `/etc/ovirt-lldp-labeler/conf.d/`. 
+
+The `ovirt-lldp-credentials.conf` contains login information to access the 
+engine API. This file is restricted with permission 0600 to prevent any
+unwanted access to the file.
+* `username` username for administrator of the engine
```

Comment:
"an administrator",  but why is an admin account required?
maybe
"username for Engine"
is more simple?